### ISSUE TITLE
`vale` CI check incorrectly fails on `INFO`-level violation in unchanged file `unknown-rt.md`

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#1180]
I cannot get the `vale` CI check to pass on any PR right now because it flags pre-existing content in `src/main/resources/org/eolang/motives/critical/unknown-rt.md` (last touched in #726, over a year ago) and `fail_on_error: true` fails the whole job even though the violation is only `INFO` severity. This isn't specific to one PR — I checked several other currently-open PRs (#1177, #1176, #1175, and my own #1178) and all of them fail on the exact same message, unrelated to what each PR actually changes.

**Steps to reproduce**

Open any PR that doesn't even touch `src/main/resources/org/eolang/motives`, and check the `vale` workflow run.

**Expected behaviour**

`vale` should only fail PRs on violations relevant to `error`-level severity (or on lines the PR actually touches), not on a pre-existing `INFO`-level style note.

**Actual behaviour**

```
{"message": "[Google.Timeless] Avoid time-based words like 'Currently' in product documentation.", "location": {"path": "src/main/resources/org/eolang/motives/critical/unknown-rt.md", "range": {"start": {"line": 3, "column": 72}}}, "severity": "INFO"}
reviewdog: input data has violations
##[error]Vale and reviewdog exited with status code: 1
```

<details>
<summary>Workflow config</summary>

```yaml
- uses: errata-ai/vale-action@2.1.2
  with:
    files: 'src/main/resources/org/eolang/motives'
    fail_on_error: 'true'
    reporter: 'github-pr-review'
    filter_mode: 'nofilter'
```
</details>

Relevant files to investigate: `.github/workflows/vale.yml`, `src/main/resources/org/eolang/motives/critical/unknown-rt.md`.
